spec/support/factory_girl.rb

```diff
@@ -14,4 +14,14 @@
     is_public true
     user
   end
+
+  factory :adventure do
+    name 'What Bar'
+    address 'Beijing, China'
+    longitude 116.40739499999995
+    latitude 39.904211
+    is_achieved false
+    notes 'An disturbingly tiny bar on the edge of the Forbidden City'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.